tools/pot/openvino/tools/pot/algorithms/quantization/fast_bias_correction/algorithm.py

```diff
@@ -202,3 +203,16 @@ def get_channel_axis(self, node):
         if isinstance(node, tuple):
             return self._channel_axis[node[0]]
         return self._channel_axis[node]
+
+    def get_inputs_shape(self, nodes_with_bias):
```

Comment:
Should we use the same method at the BiasCorrection algorithm, @IndiraSalyahova? It uses the same `get_input_shape_for_bias` for shape calculation.